pkg/setting/setting_navigation.go

```diff
@@ -0,0 +1,24 @@
+package setting
+
+import (
+	"strings"
+)
+
+func (cfg *Cfg) readNavigationSettings() error {
+	sec := cfg.Raw.Section("navigation")
+	cfg.PluginAppNavIds = map[string]string{
+		"grafana-k8s-app":                  "monitoring",
+		"grafana-synthetic-monitoring-app": "monitoring",
+		"grafana-ml-app":                   "alerts-and-incidents",
+		"grafana-incident-app":             "alerts-and-incidents",
+	}
+
+	for _, key := range sec.Keys() {
+		if strings.HasPrefix(key.Name(), "app_nav_id_") {
```

Comment:
Is this namespacing `app_nav_id_` so in the future navigation can set the location of things other than app plugins?

Reply:
yes, exactly. Could have some other navigation settings in the future

Reply:
I would suggest to use `[navigation.apps]` section instead to make things more explicit, easier to understand and configure for app plugins:

```ini
[navigation.apps]
my-app = alerting
other-app = admin
```